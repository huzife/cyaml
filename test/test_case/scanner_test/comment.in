---
- 
  - aaa: abcde
      fghijk
# comment test
  - bbb
- key1: 
        key11: ccc cccccc
        key12: ddddddd
