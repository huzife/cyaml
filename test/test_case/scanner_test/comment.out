(DOC_START)#
(BLOCK_SEQ_START)#
(BLOCK_ENTRY)#
(BLOCK_SEQ_START)#
(BLOCK_ENTRY)#
(BLOCK_MAP_START)#
(KEY)#
(SCALAR, aaa)#
(VALUE)#
(SCALAR, abcde fghijk)#
(BLOCK_MAP_END)#
(BLOCK_ENTRY)#
(SCALAR, bbb)#
(BLOCK_SEQ_END)#
(BLOCK_ENTRY)#
(BLOCK_MAP_START)#
(KEY)#
(SCALAR, key1)#
(VALUE)#
(BLOCK_MAP_START)#
(KEY)#
(SCALAR, key11)#
(VALUE)#
(SCALAR, ccc cccccc)#
(KEY)#
(SCALAR, key12)#
(VALUE)#
(SCALAR, ddddddd)#
(BLOCK_MAP_END)#
(BLOCK_MAP_END)#
(BLOCK_SEQ_END)#
